server/src/serializers/WebsiteSerializer.js

```diff
@@ -1,18 +1,23 @@
+import CommentSerializer from "./CommentSerializer.js"
+
 class WebsiteSerializer {
     static async showDetails(website) {
         const allowedAttributes = ["id", "name", "url", "description"]
-
         let serializedWebsite = {}
         for (const attribute of allowedAttributes) {
             serializedWebsite[attribute] = website[attribute]
         }
-        serializedWebsite.comments = await website.$relatedQuery("comments")
+        const relatedComments = await website.$relatedQuery("comments")
+        const serializedComments = await Promise.all(
+            relatedComments.map(async (comment) =>
+                CommentSerializer.showCommentDetails(comment))
+        )
```

Comment:
pretty sure we talked about this and explicitly discussed how we _don't_ need Promise.all! remove the promise.all AND the `async` in your map